go.mod

```diff
@@ -195,7 +195,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	helm.sh/helm/v3 v3.14.4 // indirect
-	istio.io/api v1.22.0-beta.0 // indirect
```

Comment:
are these sensible changes? reads like a move backwards

Reply:
beta.0's tag is actually older than 09344006f961. So while it looks odd "going back" to alpha, the commit used is newer. I don't know how Go does tags like this.